services/submission/src/test/java/app/coronawarn/server/services/submission/controller/FakeRequestControllerTest.java

```diff
@@ -59,10 +57,14 @@ public void setUpMocks() {
     when(fakeDelayManager.getJitteredFakeDelay()).thenReturn(1000L);
   }
 
+  // TODO Before
```

Comment:
Please remove?

Reply:
Agree with @stevesap here. Either implement this or remove. 
Another idea would be to create a helper method in this class that creates the fake header:

```java
private HttpHeaders makeCwaFakeHeader() {
  HttpHeaderBuilder.builder()
        .contentTypeProtoBufHeader()
        .cwaAuthHeader()
        .cwaFakeHeader("1")
        .build();
}
```

I think it's better than trying to extract the common scenario into the beforeEach step.